src/components/MergeRequestItem.vue

```diff
@@ -112,17 +112,13 @@
                 <span slot="badge">{{ mr.upvotes }}</span>
                 <v-icon>thumb_up</v-icon>
               </v-badge>
-              <v-tooltip top>
-                <v-btn class="mr-3" icon small
-                  slot="activator"
-                  @click.prevent="merge()"
-                  :disabled="merging || closing || mr.work_in_progress || mr.merge_status !== 'can_be_merged' || mr.state !== 'opened'"
-                  :loading="merging"
-                >
-                  <v-icon>merge_type</v-icon>
-                </v-btn>
-                <span>Merge it</span>
-              </v-tooltip>
+              <v-btn class="mr-3" small flat
```

Comment:
Sorry, maybe I did it say it well.

Instead of having a button like this:

![Capture d’écran 2019-10-10 à 15 42 48](https://user-images.githubusercontent.com/264403/66574602-cb3cbd80-eb74-11e9-886e-2220459ec793.png)

Just create a `v-list-tile` below (Like 'View on Gitlab' one for example), and put it just between icons and `View on Gitlab`